control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -672,6 +672,7 @@ func (r *Reconciler) getChannelContractResource(ctx context.Context, topic strin
 		Ingress: &contract.Ingress{
 			Host:                       receiver.Host(channel.GetNamespace(), channel.GetName()),
 			EnableAutoCreateEventTypes: feature.FromContext(ctx).IsEnabled(feature.EvenTypeAutoCreate),
+			Path:                       receiver.Path(channel.GetNamespace(), resources.MakeChannelServiceName(channel.GetName())),
```

Comment:
why not using just `receiver.Path(channel.GetNamespace(), channel.GetName())` instead of using the Service name?

Reply:
If we're using the service name in the address path for `status.address.url`, we should just use the channel name instead so that it's aligned to the Kafka Broker resource